Add apport section to docs. Relates to #1488. (#1491)
DOMjudge/domjudge: doc/manual/judging.rst

  be used to run normal DOMjudge processes; this user is only for
  internal use.

'found processes still running ... apport'
``````````````````````````````````````````

If you see error messages of the form::

  error: found processes still running as 'domjudge-run', check manually:
  2342 apport

> Then you still have ``apport`` installed and/or running. This occurs on

Reviewer: ```suggestion
Then you still have ``apport`` installed and running. This error message occurs when
```

> submitting segmentation fault solutions. Disable or uninstall the apport

Reviewer: ```suggestion
judging submissions that trigger a segmentation fault. Disable or uninstall the apport
```
